playwright/utils/common.ts

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-console */
+import { type Page } from '@playwright/test';
+
+export const acceptCookieConsentBanner = async (page: Page) => {
+  // Check if the cookie banner is present on the page
+  const cookieBanner = page.locator('#ez-cookie-notification');
+  const isBannerVisible = await cookieBanner.isVisible();
+
+  if (isBannerVisible) {
+    // Click the accept button on the cookie banner
+    const acceptButton = page.locator('#ez-cookie-notification__accept');
+
+    await acceptButton.click();
+    console.log('Cookie banner accepted');
+  } else {
+    console.log('No cookie banner found');
```

Comment:
Personal opinion, but I don't think we need the else console log as it spams the console.

<img width="1065" alt="Screenshot 2024-10-24 at 12 12 43" src="https://github.com/user-attachments/assets/0b422cbf-3528-4f68-963f-77ea44828c52">

Reply:
I've removed console logs, rebased the branch and updated the tests. thanks for your time for reviewing this again )